Source/WebCore/platform/text/cocoa/LocalizedDateCache.mm

```diff
@@ -133,13 +131,14 @@ static void _localeChanged(CFNotificationCenterRef, void*, CFStringRef, const vo
 
 static float calculateMaximumWidthForWeek(const MeasureTextClient& measurer)
 {
-    std::array<float, 10> numLengths = { };
-    for (int i = 0; i < 10; i++)
+    std::array<float, 10> numLengths;
+    for (int i = 0; i < 10; ++i)
         numLengths[i] = measurer.measureText([NSString stringWithFormat:@"%d", i]);
```

Comment:
Not for this change, but there is no reason for these to be going through NSString. measureText() takes a WTF::String.